GoogleUtilities/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 7.0.0
+- All APIs are now public. All CocoaPods private headers are tranitioned to public.
```

Comment:
transitioned

Reply:
Do we want to leave a comment mentioning that it's still not for non-Google usage?